src/full/Agda/Interaction/Library.hs

```diff
@@ -181,21 +181,25 @@ findProjectConfig'
   :: FilePath                          -- ^ Candidate (init: the directory Agda was called in)
   -> LibErrorIO ProjectConfig          -- ^ Actual root and @.agda-lib@ files for this project
 findProjectConfig' root = do
-  getCachedProjectConfig root >>= \case
-    Just conf -> return conf
-    Nothing   -> do
-      libFiles <- liftIO $ filter ((== ".agda-lib") . takeExtension) <$> getDirectoryContents root
-      case libFiles of
-        []     -> liftIO (upPath root) >>= \case
-          Just up -> do
-            conf <- findProjectConfig' up
+  root <- liftIO (canonicalizePath root)
+  if root == "/nix/store"
```

Comment:
Is this absolute path OS-agnostic?

Reply:
Nix doesn't support any OSes that use a different separator than `/`, but it does support running in a custom prefix, e.g. `~/.nix/store`. There's no hope to cover all such possible configurations, so I think this is good enough.